tests/subyt/test_conditional.py

```
@@ -1,96 +1,152 @@
+        # process
+        log.debug(f"processing test-template: {tpl_path / tlp_name}")
+        try:
+            g.process(
+                tlp_name,
+                inputs,
+                generator_settings,
+                sink,
+                vars_dict={"my_domain": "realexample.org"},
+            )
+        except Exception as e:
+            log.exception(e)
```

Ruff (0.8.2)

Redundant exception object included in `logging.exception` call

(TRY401)